sys/sched_round_robin/sched_round_robin.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (C) 2021 TUBA Freiberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @ingroup     sys
+ * @{
+ *
+ * @file
+ * @brief       Round Robin Scheduler implementation
+ *
+ * @author      Karl Fessel <karl.fessel@ovgu.de>
+ *
+ * @}
+ */
+
+#include "sched.h"
+#include "thread.h"
+#include "xtimer.h"
+#include "sched_round_robin.h"
+
+#define ENABLE_DEBUG 0
+#include "debug.h"
+
+static void sched_round_robin_cb(void *d);
+
+static xtimer_t rr_timer = { .callback = sched_round_robin_cb };
+
+/*
+ * assuming simple reads from and writes to a byte to be atomic on every board
+ */
+static uint8_t current_rr_priority;
+
+void sched_runqueue_change_cb(uint8_t prio);
+
+void sched_round_robin_cb(void *d)
```

Comment:
why "d"?

Reply:
it need a name and it is unused any recommendation?

Reply:
`arg`, but I admit this is opinionated. :)

Reply:
Somehow i think of `arg` would be something i want to parse (list / buffer).
But i also have no real connection to `d` aside from it is an unused single letter name (making it very local) and it is not i j k c e n,  which are single letter names i uses oftenfor traversing list buffers or counting (local only).
I did some searching and  found that some people use `_` to make unused, but i think this would make other people assume that these are statics/privates/local/internal only.